Sources/_RegexParser/Regex/Parse/LexicalAnalysis.swift

```diff
@@ -767,8 +767,10 @@ extension Source {
         return opt
 
       // Swift semantic level options
-      case "X": return advanceAndReturn(.graphemeClusterSemantics)
-      case "u": return advanceAndReturn(.unicodeScalarSemantics)
+      // These are disabled for now until we figure out the right semantics for
+      // them
+      // case "X": return advanceAndReturn(.graphemeClusterSemantics)
+      // case "u": return advanceAndReturn(.unicodeScalarSemantics)
```

Comment:
We could still parse them, but mark them unsupported in Sema (see `validateMatchingOption`), what do you think?